DAO/project/contracts/DAO-contract/src/main.sw

```diff
@@ -155,18 +196,35 @@ impl DaoVoting for Contract {
 
         let mut proposal = storage.proposals.get(proposal_id).try_read().unwrap();
         require(!proposal.executed, ProposalError::ProposalExecuted);
-        require(proposal.deadline < height().as_u64(), ProposalError::ProposalStillActive);
+        require(
+            proposal
+                .deadline < height()
+                .as_u64(),
+            ProposalError::ProposalStillActive,
+        );
 
         // TODO figure out how to prevent approval percentage from overflowing
         // When close to the u64 max
         // https://github.com/FuelLabs/sway-applications/issues/106
         let acceptance_percentage = proposal.yes_votes * 100 / (proposal.yes_votes + proposal.no_votes);
-        require(proposal.acceptance_percentage <= acceptance_percentage, ProposalError::InsufficientApprovals);
+        require(
+            proposal
+                .acceptance_percentage <= acceptance_percentage,
+            ProposalError::InsufficientApprovals,
+        );
 
         proposal.executed = true;
         storage.proposals.insert(proposal_id, proposal);
 
-        asm(call_data: proposal.proposal_transaction.call_data, amount: proposal.proposal_transaction.amount, asset: proposal.proposal_transaction.asset, gas: proposal.proposal_transaction.gas) {
+        asm(call_data: proposal
```

Comment:
The formatting here is faulty imo. I would raise an issue.
Likewise, in `asm` blocks below.